sre_maturity.md

```diff
@@ -0,0 +1,115 @@
+It doesn't have to be completely representative of what users expect from your service, but it forms the building blocks of an SLO review process and improving on them in time.
+Starting with a lower SLO than where you want to get to sets expectations early.
+
+#### “Admin” access to the service
+
+Your team has the level of access required to fix production problems.
+If not, you can’t expect them to debug and fix problems in a timely manner.
+Being overly controlling of access shouldn’t be a priority at this phase.
+That can come later with things like audited access, permissions escalation & tiered access as you mature the debugging experience of your service.
+You should trust your team and assume positive intent.
+
+#### On Call, Escalations and Incidents
+
+You have a team with an on call schedule.
+It doesn’t have to be 24/7. It doesn’t have to be automated.
+But there must be someone responsible to respond when things go bad.
+Setting expectations at this early stage with a reduced SLO means you can tailor the schedule to more reasonable times for your team.
+
+Following on from this, you have an escalation process. A way to be notified of problems. 
+It should be possible to raise a problem with the SRE team and the people that build the service, like engineers, QE, writers.
+This can be a mailing list, slack channel, issue system, whatever.
+Formalise it and share it.
+
+If things go really bad, you write up an RCA.
+You should be starting to build up a library of RCAs that can feed into doing more proactive things as you mature e.g. Incident rehearsals.
+Each RCA includes a 'blameless' post mortem session with the people involved while the incident is still fresh in their head.
+This post mortem feeds important reliability improvements back to the team.
+
+#### Room to improve
+
+Your team has sufficient time in their day to pursue improvement.
+Your team needs room to practice processes, improve them and automate things.
+Without this room, your team will mature more slowly and get bogged down in just keeping the service running day to day.
+This can lead to scaling problems, both for the service and the team.
+It isn't necessary to formally measure how much time is spent on keeping things running vs. feature development or automation.
+That can come later, however you are familiar with the concept of Toil.
+
+
+## Phase 2: The ‘Walk’ Phase (WIP)
+
+### Overview
+
+Your team is well established, has working processes and is familiar with SRE principles.
+A lot of those principles are baked into their processes & daily work.
+In this phase your team is building on the foundations of observability and starting to reflect on their learnings about the service. 
+Feedback loops, automation and 'shifting left' with different types of testing are the main theme of this phase.
+Your team is capable of scaling the service and number of users in a profitable way.
```

Comment:
Your team is capable of scaling the service.  drop the last half

Reply:
```suggestion
Your team is capable of scaling the service.
```